src/components/PokeCard.js

```diff
@@ -50,8 +50,9 @@ const PokeCard = ({
   };
 
   useEffect(() => {
-    // pokemon modal content
+    // get pokemon modal content
     triggerModalData();
+    // eslint-disable-next-line
```

Comment:
what does the eslint-disable-next-line do?

Reply:
Removes the warnings from EsLint in terminal. They were unuseful since putting the function in dependency array of those `useEffects` could cause too many re-renders.